source/ptreferential/ptreferential_ng.cpp

```diff
@@ -263,6 +263,13 @@ struct Eval : boost::static_visitor<Indexes> {
             }
         } else if (type == Type_e::VehicleJourney && f.method == "has_disruption" && f.args.empty()) {
             indexes = get_indexes_by_impacts(type::Type_e::VehicleJourney, data);
+        } else if (type == Type_e::VehicleJourney && f.method == "has_active") {
+            // useful only for VJ for vehicle_positions api
+            const auto rt_level =
+                (type == Type_e::VehicleJourney) ? rt_level_from_string(f.args.back()) : type::RTLevel::Base;
```

Comment:
I fear using `f.args.back()` will cause problem in the future if we add more arguments to the function.
Maybe use the index of the argument instead of `back()` ?

Reply:
ok